dv/cosim/cosim_dpi.h

```diff
@@ -19,6 +19,7 @@ void riscv_cosim_set_mip(Cosim *cosim, const svBitVecVal *mip);
 void riscv_cosim_set_nmi(Cosim *cosim, svBit nmi);
 void riscv_cosim_set_debug_req(Cosim *cosim, svBit debug_req);
 void riscv_cosim_set_mcycle(Cosim *cosim, svBitVecVal *mcycle);
+void riscv_cosim_set_csr(Cosim *cosim, int csr_id, svBitVecVal *csr_val);
```

Comment:
I may be out of my depth, but is it possible for `csr_val` to be a const pointer? Or is that not expressible in the .svh file?